database/migrations/2020_08_26_162836_create_h5p_tags_table.php

```diff
@@ -16,7 +16,6 @@ public function up()
         Schema::create('h5p_tags', function (Blueprint $table) {
             $table->id();
             $table->string('name', 31);
-            $table->timestamps();
```

Comment:
Why is this removed? It is the auto generic columns and also nullable so it will be no problem.
Please add it again.

Reply:
no needed. not in original tables form H5P.

Reply:
Add it.